images/build/Dockerfiles/Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM oryxdevmcr.azurecr.io/public/oryx/build:lts-versions-debian-stretch AS main
 # Build run script generators (to be used by the `oryx run-script` command)
 ###
 # DisableDockerDetector "Below image not yet supported in the Docker Hub mirror"
-FROM golang:1.15-stretch as startupScriptGens
+FROM golang:1.18-stretch as startupScriptGens
```

Comment:
Is the discrepancy between golang 1.18 and [1.19](https://github.com/microsoft/Oryx/pull/1957/files#diff-fd77fd00771c313a727f3c931eb29e02bda69ae692c91b5677e560aef0ef17a0R28) intentional?